modules/appeals_api/spec/factories/supplemental_claims.rb

```diff
@@ -23,16 +23,6 @@
     trait :status_submitted do
       status { 'submitted' }
     end
-
-    trait :completed_a_day_ago do
-      updated_at { 1.day.ago }
```

Comment:
Trying to change the `updated_at` in this way doesn't work (and had workarounds in each test that used the trait) so I removed the traits entirely.